### ISSUE TITLE
[v2] ImportError for tzset on Windows

### ISSUE THREAD

**Issue by browniebroke** [adamchainz/time-machine#100]
In one of my project, I received an automatic update to update time-machine to the newly published v2 and my CI crashes on Windows (Python 3.7, 3.8 and 3.9) with the following error:

```
ImportError: cannot import name 'tzset' from 'time' (unknown location)
```

I don't own a Windows machine, but one can check the logs here: https://github.com/browniebroke/netlify-builds/pull/27/checks?check_run_id=1716666697#step:6:52

Not really urgent or anything, but I thought I report the problem. Is it a missing dependency or something different with the stdlib?

**Comment by adamchainz** [comment]
Fixed in version 2.0.1.

**Comment by browniebroke** [comment]
Confirmed as all fixed! Thanks for quick release!
